pyproject.toml

```
@@ -25,6 +25,7 @@ packages = [
 python = ">=3.5"
 Django = ">=2.2"
 pyodbc = ">=4.0"
+pytz
```

I don't think this is valid.

poetry is giving me `Invalid TOML file /home/kadler/projects/django-ibmi/pyproject.toml: Unexpected character: '\n' at line 28 col 4` I think it should be this instead:

```suggestion
pytz = "*"
```